app/services/get-case-progress.js

```diff
@@ -1,14 +1,14 @@
 const getBreadcrumbs = require('./get-breadcrumbs')
 const getOrganisationUnit = require('./helpers/org-unit-finder')
-const getCaseProgress = require('./data/get-individual-caseload-progress')
+const getCaseProgress = require('./data/get-caseload-progress')
 
 module.exports = function (id, organisationLevel) {
   var result = {}
   var organisationalUnitType = getOrganisationUnit('name', organisationLevel)
-  var caseProgressPromise = getCaseProgress(id)
+  var caseProgressPromise = getCaseProgress(id, organisationLevel)
 
   return caseProgressPromise.then(function (results) {
-    result.caseProgress = results
+    result.caseProgressList = [results]
```

Comment:
Does this need to be wrapped in `[]`? It will now be returning an Array, so can the Array itself not be used?